examples/flutter_gallery/lib/demo/fortnightly/fortnightly.dart

```diff
@@ -0,0 +1,232 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class FortnightlyDemo extends StatelessWidget {
+  static const String routeName = '/fortnightly';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Typography Demo',
+      theme: _fortnightlyTheme,
+      home: Scaffold(
+        body: Stack(
+          children: <Widget>[
+            FruitPage(),
+            Padding(
+              padding: EdgeInsets.only(top: MediaQuery.of(context).padding.top),
+              child: ShortAppBar(
+                onBackPressed: () {
+                  Navigator.pop(context);
+                },
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class ShortAppBar extends StatelessWidget {
+  const ShortAppBar({this.onBackPressed});
+
+  final VoidCallback onBackPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 96,
+      height: 50,
+      child: Material(
+        elevation: 4,
+        shape: const BeveledRectangleBorder(
+          borderRadius: BorderRadius.only(bottomRight: Radius.circular(22)),
+        ),
+        child: Row(
+          children: <Widget>[
+            IconButton(
+              icon: const Icon(Icons.arrow_back),
+              onPressed: onBackPressed,
+            ),
+            const SizedBox(width: 12),
+            Image.asset(
+              'logos/fortnightly/fortnightly_logo.png',
+              package: 'flutter_gallery_assets',
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class FruitPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final TextTheme textTheme = Theme.of(context).primaryTextTheme;
+
+    return SingleChildScrollView(
+      padding: EdgeInsets.only(bottom: MediaQuery.of(context).padding.bottom),
```

Comment:
You should use SafeArea here and advertise as much in the README (SafeArea is preferable to using MediaQuery).

Reply:
Done!